functions/uploads.py

```diff
@@ -55,19 +72,14 @@ def ingest_upload(event: dict, context: BackgroundContext):
 
```

Comment:
as you're already moving prints around, I'd add something here as well, to mark the start of fn exec.